src/core_modules/capture-core/converters/clientToForm.js

```diff
@@ -70,6 +83,8 @@ const valueConvertersForType = {
     [dataElementTypes.TRUE_ONLY]: () => 'true',
     [dataElementTypes.BOOLEAN]: (rawValue: boolean) => (rawValue ? 'true' : 'false'),
     [dataElementTypes.AGE]: convertAgeForEdit,
+    [dataElementTypes.NUMBER_RANGE]: convertNumberRangeForDisplay,
+    [dataElementTypes.DATE_RANGE]: convertDateRangeForDisplay,
```

Comment:
Hey, 

There are a couple of  `*_RANGE` TEAs missing like: 
``` 
    INTEGER_RANGE 
    INTEGER_POSITIVE_RANGE, 
    INTEGER_NEGATIVE_RANGE, 
    INTEGER_ZERO_OR_POSITIVE_RANGE, 
    TIME_RANGE 
``` 
The full list supported by the search component can be found in the [searchFormElementConverter.js](https://github.com/dhis2/capture-app/blob/master/src/core_modules/capture-core/components/Pages/Search/SearchForm/SearchFormElementConverter/SearchFormElementConverter.js#L51). Does it make sense to include them here?  

Thanks!

Reply:
hey, it seems like I can not find the way to add these *_RANGE TEAs via maintenance  app. Any chance that they are no longer supported? 

<img width="727" alt="Screenshot 2022-10-07 at 08 28 18" src="https://user-images.githubusercontent.com/89806888/194497516-9ff78635-ee39-4b93-ad46-892c0884de3a.png">

Reply:
we discussed this offline and I updated the converter to reflex the changes.